src/_guides/api-basics/dev-guide-api-design-first.md

```diff
@@ -1,138 +1,128 @@
 ---
-title: A Developer's Guide to API-First Design
-authors: Alex Doukas
+title: A Developer's Guide to API Design-First
+authors: phil
 image: images/guides/api-first-design-guide.png
 canonical_url: https://bump.sh/blog/dev-guide-api-design-first
-excerpt: Learn about the principles of API-first design and how it can benefit your organization.
-date: 2024-03-25
+excerpt: Learn about the principles of API design-first and how it can benefit your organization.
+date: 2024-11-15
 ---
 
-API-first design is a software development approach built around the idea that the application programming interfaces (APIs) should be the primary focus of the development process, with other system components, such as the user interface (UI) being developed later.
+API Design-First, also called schema-first or contract-first, is all about designing the interface of an API before writing any code. It's about planning the [API contract](https://bump.sh/blog/api-contracts-extended-introduction) first, and defining what the API does and how it works so everyone's on the same page before implementation starts. This approach has been around for a while, and over time, it's evolved to meet the needs of different technologies. These days, OpenAPI has become the defacto standard for designing REST APIs, and it can make life easier for everyone the whole way through the lifecycle of an API.
```

Comment:
This section of our guides is usually "specification-agnostic". If we mention OpenAPI (which can make sense), I would bring some balance with mentioning other technologies such as AsyncAPI.